### PR TITLE
Bug/homepage links

### PR DESCRIPTION
**Pull request by natyork**
Conditionally render links on homepage and header that will produce a 500 error if user is not logged in.

Per story rejection comments on:
https://www.pivotaltracker.com/story/show/141187843

### CODE REVIEW

File: app/views/welcome/_next-event.html.erb [hunk 1]
```diff
@@ -18,16 +18,14 @@
     <% else %>
       <p>
         There are currently no presentations for this event.
-        <%= link_to 'Sign up', new_presentation_path %>
+        <%= render 'next-event-links' %>
```
Comment (graygilmore) [inline app/views/welcome/_next-event.html.erb]: Maybe we could rename this partial to `new_presentation_link` or something like that? That way we could use it in other areas of the system, too.

File: app/views/welcome/_next-event.html.erb [hunk 2]
```diff
@@ -18,16 +18,14 @@
...
         to teach some people something!
       </p>
     <% end %>
 
   <% else %>
     <p>
       There are no upcoming events.
-      <%= link_to 'Create one', new_event_path %>
```
Comment (jacquesporveau) [inline app/views/welcome/_next-event.html.erb]: With this being removed you can probably have that `<p>` tag be a one line.

File: app/views/welcome/_next-event.html.erb (hunk 1 above)
Comment (jacquesporveau) [inline app/views/welcome/_next-event.html.erb]: I think this is fine for now but maybe a hint for the user that they have to sign in to create one in the future. There are currently no presentations for this event... sign in. Feels weird. I like There are currently no presentations for this event... sign in to create one. Either is fine really.

**Comment by natyork** [comment]
Updated per @graygilmore & @jacquesporveau comments

**Review by jacquesporveau — COMMENTED** [review]
LGTM

**Review by qr8r — APPROVED** [review]

**Pull request merged**